src/libraries/System.Linq/tests/Shuffler.cs

```diff
@@ -14,6 +14,6 @@ public static T[] Shuffle<T>(T[] array)
             return array;
         }
 
-        public static IEnumerable<T> Shuffle<T>(this IEnumerable<T> source) => Shuffle(source.ToArray());
+        public static IEnumerable<T> Shuffle<T>(IEnumerable<T> source) => Shuffle(source.ToArray());
```

Comment:
Can't we just get rid of this now?

Reply:
Possibly, but the tests were currently deterministic, and removing this and changing all call sites to use the new one would make them not.

Reply:
Seems like another argument in favor of a deterministic overload?